src/Sylius/Bundle/PromotionBundle/spec/Form/Type/CouponGenerateInstructionTypeSpec.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace spec\Sylius\Bundle\PromotionBundle\Form\Type;
+
+use PhpSpec\ObjectBehavior;
+use Prophecy\Argument;
+use Symfony\Component\Form\FormBuilder;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * @author Arkadiusz Krakowiak <arkadiusz.krakowiak@lakion.com>
+ */
+class CouponGenerateInstructionTypeSpec extends ObjectBehavior
```

Comment:
I guess it can be removed as specifying form types isn't much helpful or beneficial.